streams/src/test/java/org/apache/kafka/streams/kstream/KStreamBuilderTest.java

```diff
@@ -22,12 +22,23 @@
 import org.apache.kafka.streams.errors.TopologyBuilderException;
 import org.apache.kafka.test.KStreamTestDriver;
 import org.apache.kafka.test.MockProcessorSupplier;
+import org.junit.After;
 import org.junit.Test;
 
 import static org.junit.Assert.assertEquals;
 
 public class KStreamBuilderTest {
 
+    private KStreamTestDriver driver = null;
+
+    @After
+    public void cleanup() {
+        if (driver != null) {
+            driver.close();
+        }
+        driver = null;
+    }
```

Comment:
It is possible, but you'd need to make a few changes to `KStreamTestDriver` , i.e., everything currently in the constructor would need to be moved into a method that can be called from the tests. So you'd end up having an overloaded `initiliaze` method instead of the various constructors.